samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -1372,9 +1387,10 @@ static int cmd_context(const struct shell *shell, size_t argc, char **argv)
 
 static int location_find(const struct shell *shell, const char *name)
 {
-	for (size_t i = 0; i < ARRAY_SIZE(locations); i++) {
-		if (strcasecmp(locations[i].name, name) == 0) {
-			return locations[i].location;
+
+	for (int i = -1; i < get_bits_set_num(BT_AUDIO_LOCATION_ANY); i++) {
+		if (strcasecmp(bt_audio_location_bit_to_str(BIT(i)), name) == 0) {
+			return i + 1;
 		}
 	}
```

Comment:
I'm not getting the -1 and then +1 operation, could you explain why it doesn't go from 0? And if there is a match for `i`, why do we need to add 1 before returning? If name is "Back Left" (BIT(4)) for instance, then `i` would be 4 and the return value is 5, but it should really be BIT(4) which is 16. I guess that was made to work for Mono Audio as this is 0 and thus outside of the whole BIT() operation. I suggest to add a special case for that outside of the for loop:

```suggestion
	/* Check if name == Mono Audio */
	if (strcasecmp(bt_audio_location_bit_to_str(BT_AUDIO_LOCATION_MONO_AUDIO), name) == 0) {
	        return BT_AUDIO_LOCATION_MONO_AUDIO;
	}

	for (int i = 0; i < get_bits_set_num(BT_AUDIO_LOCATION_ANY); i++) {
		if (strcasecmp(bt_audio_location_bit_to_str(BIT(i)), name) == 0) {
			return BIT(i);
		}
	}
```